### DISCUSSION TITLE
字体大小，组件padding 大小优化建议

### DISCUSSION THREAD

**Discussion by melin** [ant-design/ant-design-pro#10278] [Ideas]
antd design pro一般用在后台管理系统，后台管理系统文本字体大小一般12px，标题为14-16px，组件padding 也比较小，这样看起来比较紧凑精致，展现的内容也比较多。太大的字体和padding 做出来的产品显得很粗糙。可以借鉴一下extjs，easyui 默认的大小。 现在很多组件默认很大，包括layui, antd，ElementUI。很多时候前端开发不调整(也不关系最终效果，只关系功能)，做出来的效果很难看。
建议layui 字体大小和组件padding 进行优化。或者提供一个开箱即用的紧凑版样式。

个人意见仅供参考，参考阿里云的一些产品，文字基本都是12px。
dataworks 是阿里云产品
dataleap 是字节火山引擎产品
dataarts studio 是华为云产品
![image](https://user-images.githubusercontent.com/1145830/194733014-66aaa839-3bfc-4bbd-97eb-91384e660ba3.png)

这个页面如果14px 字体，会感觉比较粗糙：
![image](https://user-images.githubusercontent.com/1145830/194733020-98ff2da2-1964-475b-94a4-f182fb3a9376.png)

**Comment by hihuangwei** [comment]
原来是12 后来调到14
